telemetry/specs/INT.mdk

```diff
@@ -384,12 +384,32 @@ ignore the designated bit position.)
 * TCP/UDP destination Port field: A port number in layer 4 will be reserved to
 indicate the existence of INT after TCP/UDP. INT source changes the destination
 Port, and sink must restore the original port number.
+* New Probe Marker fields: arbitrary 32-bit values are inserted after IP TCP/UDP header
+to indicate the existence of INT after TCP/UDP. These fileds should be interpreted
+as unsigned integer values and are initialized to a configured value. 
```

Comment:
Can we say that the value is also per-domain decision?

Reply:
What's the purpose of mentioning "unsigned integer values"?

Reply:
The "per-domain" was mentioned in below context.
For the "integer" just added context "These fileds should be interpreted
as unsigned integer values, stored in network byte order"